packages/esm-patient-medications-app/src/add-drug-order/add-drug-order.workspace.tsx

```diff
@@ -55,6 +60,10 @@ export default function AddDrugOrderWorkspace({
     (finalizedOrder: DrugOrderBasketItem) => {
       finalizedOrder.careSetting = careSettingUuid;
       finalizedOrder.orderer = session.currentProvider.uuid;
+      //Setting dateActivated ensures that the order date is accurately captured, which is essential for RDE
+      if (visitEndDate) {
+        finalizedOrder.dateActivated = visitStartDate;
```

Comment:
In RDE, this should always be a user-entered date / time (which we can _default_ to the visit start date or w/e).